### ISSUE TITLE
Create sometimes does not include header

### ISSUE THREAD

**Issue by coinop-logan** [gianlucaschoefer/gianlucaschoefer.github.com#9]
Unfortunately I don't know how to reliably recreate this bug, but sometimes, the create page will load without a header. Usually a refresh brings it back. No idea what's causing it.

**Comment by coinop-logan** [comment]
This happens on other pages as well.

**Comment by coinop-logan** [comment]
https://i.imgur.com/F2fgeSC.png

**Comment by gianlucaschoefer** [comment]
Could you please check what the developer console says next time? @coinop-logan

**Comment by coinop-logan** [comment]
Yeah, thought of that afterwards. I'll do so next time.

**Comment by coinop-logan** [comment]
No output from the developer console: https://i.imgur.com/sqO68gu.png

**Comment by gianlucaschoefer** [comment]
I didn't encounter this not a single time. Very weird, will have to test different environments. Are you using chrome ?

**Comment by coinop-logan** [comment]
Yes, Chrome. It only happens about 10% of the time. Looks like the <div id="nav-placeholder"></div> element is never replaced when it does happen.

**Comment by coinop-logan** [comment]
It also seems to only happen once, then any further refreshes always have it show up correctly. I'll keep an eye out for more patterns and let you know.

**Comment by gianlucaschoefer** [comment]
Tried a few things, will continue to observe..
